recipe/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = "2.12.0" %}
+{% set name = "tensorflow-text" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tensorflow/text
+  git_rev: v{{ version }}
```

```suggestion
  url: https://github.com/tensorflow/text/archive/v{{ version }}.tar.gz
  sha256: 0de6d94d0b76a6050ae8c0ff618bbc5d9a646ef9929161149aad63974f8a56f2
```
@boldorider4 You can create a recipe by grayskull and check it works:
`grayskull pypi https://github.com/tensorflow/text --tag v2.12.0`

---

thanks